HCaptcha/Classes/HCaptcha.swift

```diff
@@ -139,9 +161,16 @@ public class HCaptcha {
     public convenience init(
         apiKey: String? = nil,
         baseURL: URL? = nil,
-        endpoint: Endpoint = .default,
         locale: Locale? = nil,
-        size: Size = .invisible
+        size: Size = .invisible,
+        rqdata: String? = nil,
+        sentry: Bool = false,
+        apiEndpoint: URL = URL(string: "https://hcaptcha.com/1/api.js")!,
```

Comment:
```suggestion
        apiEndpoint: URL = URL(string: "https://js.hcaptcha.com/1/api.js")!,
```

(and note rename comment about apiEndpoint above)

Reply:
Done